ci/docker_ci.sh

```diff
@@ -197,6 +198,11 @@ build_and_maybe_push_image () {
     docker_image_tarball="${ENVOY_DOCKER_IMAGE_DIRECTORY}/envoy${image_type}.tar"
 
     if ! is_windows; then
+        # `--sbom` and `--provenance` args added for skopeo 1.5.0 compat,
+        # can probably be removed for later versions.
+        args+=(
+            "--sbom=false"
+            "--provenance=false")
```

Comment:
These args will disable provenance and sbom attestations. But it should be ok for this fix. 🤔

Reply:
also its a very new feature https://www.docker.com/blog/generate-sboms-with-buildkit/

the only reason we are getting it is because we install `buildx` latest

Reply:
just confirming we dont currently use it ...

```console
$ docker buildx imagetools inspect envoyproxy/envoy:v1.25-latest --format "{{json .SBOM.SPDX}}"
null
$ docker buildx imagetools inspect envoyproxy/envoy:dev --format "{{json .SBOM.SPDX}}"
null
```

Reply:
This is very possbile. These is even no docker sbom comand in my local env.